### PR TITLE
Lint and formatting fixes

### PR DESCRIPTION
**Pull request by Dev-Khant**
## Description

Lint and formatting fixes

## Type of change

- [x] Refactor (does not change functionality, e.g. code style improvements, linting)

## How Has This Been Tested?

- [x] Unit Test

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] My changes generate no new warnings
- [x] New and existing unit tests pass locally with my changes
- [x] I have checked my code and corrected any misspellings

### CODE REVIEW

**Comment by Dev-Khant** [comment]
@taranjeet Here are the lint and formatting fixes.

File: embedchain/bots/__init__.py [hunk 1]
```diff
@@ -1,4 +1,3 @@
-from embedchain.bots.poe import PoeBot
-from embedchain.bots.whatsapp import WhatsAppBot
-# TODO: fix discord import
-# from embedchain.bots.discord import DiscordBot
\ No newline at end of file
+from embedchain.bots.poe import PoeBot  # noqa: F401
+from embedchain.bots.whatsapp import WhatsAppBot  # noqa: F401
+from embedchain.bots.discord import DiscordBot  # noqa: F401
```
Comment (taranjeet) [inline embedchain/bots/__init__.py:3]: this needs to be commented. There is an issue with the import.
If the comment is throwing lint error, please let me know here first.
@Dev-Khant

File: embedchain/bots/whatsapp.py [hunk 2]
```diff
@@ -3,6 +3,9 @@
 import signal
 import sys
 
+from flask import Flask, request
```
Comment (taranjeet) [inline embedchain/bots/whatsapp.py:6]: we cannot do this.
This is done on purpose.

Comment (taranjeet) [reply embedchain/bots/whatsapp.py:6]: we need to figure out a way to ignore this.
imports needs to be lazily loaded in the __init__

File: embedchain/bots/__init__.py (hunk 1 above)
Comment (Dev-Khant) [reply embedchain/bots/__init__.py:3]:
```
from .EmbeddingFunctions import EmbeddingFunctions  # noqa: F401
from .Providers import Providers  # noqa: F401
from .VectorDatabases import VectorDatabases  # noqa: F401
from .VectorDimensions import VectorDimensions  # noqa: F401

```

I did it on purpose by looking at `embedchain\models\__init__.py`. Let me know if it needs to be commented

File: embedchain/bots/whatsapp.py (hunk 2 above)
Comment (Dev-Khant) [reply embedchain/bots/whatsapp.py:6]: Okay will see how it can be done

File: embedchain/bots/__init__.py (hunk 1 above)
Comment (taranjeet) [reply embedchain/bots/__init__.py:3]: The first and second line should stay.

**Comment by Dev-Khant** [comment]
@taranjeet Please have a look

**Comment by Dev-Khant** [comment]
Its fixed

**Comment by cachho** [comment]
@Dev-Khant when I run `make format lint` I pickup a lot more. Can you include that?

**Review by taranjeet — APPROVED** [review]

**Comment by taranjeet** [comment]
looks good to me.

**Review by taranjeet — APPROVED** [review]
looks good to me.

**Pull request merged**